python/lsst/analysis/tools/actions/plot/propertyMapPlot.py

```diff
@@ -0,0 +1,443 @@
+        **kwargs,
+    ) -> Mapping[str, Figure]:
+        """Make the survey property map plot.
+
+        Parameters
+        ----------
+        data : `KeyedData`
+            The HealSparseMap to plot the points from.
+
+        tract: `lsst.skymap.tractInfo.ExplicitTractInfo`
+            The tract info object.
+
+        plotInfo : `dict`
+            A dictionary of information about the data being plotted.
+
+        **kwargs :
+            Additional keyword arguments to pass to the plot.
+
+        Returns
+        -------
+        figDict : `dict` [`~matplotlib.figure.Figure`]
+            The resulting figures.
+        """
+
+        figDict: dict[str, Figure] = {}
+
+        # Plotting customization.
+        colorbarTickLabelSize = 14
+        colorBarAspect = 16
+        nBinsHist = 100
+        rcparams = {
+            "axes.labelsize": 18,
+            "axes.linewidth": 1.8,
+            "xtick.labelsize": 13,
+            "ytick.labelsize": 13,
+        }
+        plt.rcParams.update(rcparams)
+
+        # Muted green, red and blue for labeling the full and zoomed-in maps.
+        zoomColors = ["#265D40", "#8B0000", "#00008B"]
+
+        for mapName, mapData in data.items():
+            fig, ((ax1, ax2), (ax3, ax4)) = plt.subplots(2, 2, figsize=(16, 16))
+
+            # Reduce whitespace but leave some room at the top for `plotInfo`.
+            plt.subplots_adjust(left=0.064, right=0.96, top=0.855, bottom=0.07, wspace=0.18, hspace=0.24)
+
+            # Get the values for the valid pixels of the full tract.
+            values = mapData[mapData.valid_pixels]
+            goodValues = np.isfinite(values)
+            values = values[goodValues]  # As a precaution.
+
+            # Make a concise human-readable label for the plot.
+            plotInfo["coaddName"] = mapName.split("Coadd_")[0]
+            plotInfo["operation"] = self.getLongestSuffixMatch(
+                mapName, ["min", "max", "mean", "weighted_mean", "sum"]
+            )
+            plotInfo["property"] = mapName[
+                len(f"{plotInfo['coaddName']}Coadd_") : -len(plotInfo["operation"])
+            ].strip("_")
+            label = plotInfo["property"].replace("_", " ").title().replace("Psf", "PSF")
+
+            fullExtent = None
+            zoomIdx = []
+            for ax, zoom, zoomFactor, zoomColor in zip(
+                [ax1, ax3, ax4], [True, False, False], [None, *zoomFactors], zoomColors
+            ):
+                extent = self.getZoomedExtent(fullExtent, zoomFactor)
+                sp = skyproj.GnomonicSkyproj(
+                    ax=ax,
+                    lon_0=tract.ctr_coord.getRa().asDegrees(),
+                    lat_0=tract.ctr_coord.getDec().asDegrees(),
+                    extent=extent,
+                    rcparams=rcparams,
+                )
+                sp.draw_hspmap(mapData, zoom=zoom)
+                sp.set_xlabel("RA (deg)")
+                sp.set_ylabel("Dec (deg)")
```

Comment:
Oh another thing: `(deg)` is redundant because the tick labels all have the units.